pkg/generated/crds/crds_generate.go

```diff
@@ -0,0 +1,117 @@
+// +build ignore
```

Comment:
This LGTM - only thing I'm wondering is if we want to relocate this file to be outside of `pkg/generated`? It could make sense to go in a subdir of `hack`.

Reply:
Yeah, I was thinking about that too - I think that makes sense, will move it there.

Reply:
I think this was moved to `hack/crd-gen/main.go` and this needs to be deleted?

Reply:
yep 😅thanks for the catch!